packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -203,7 +203,12 @@ export abstract class SfdxCommandletExecutor<T>
     const cancellationToken = cancellationTokenSource.token;
 
     const execution = new CliCommandExecutor(this.build(response.data), {
-      cwd: vscode.workspace.rootPath
+      cwd: vscode.workspace.rootPath,
+      stdio: [
+        'pipe',
+        'pipe', // fs.openSync(path.join(vscode.workspace.rootPath!, 'stdout.txt'), 'w'),
```

Comment:
A possible, but annoying workaround, is to pipe the result to another stream.
Using a temp file seems to gather all the data. 
However, with that, you lose the ability to stream data as it arrives.